DOCS-3003: Add prerequisite of physically assembling hardware to components and services (#3540)
viamrobotics/docs: docs/components/movement-sensor/adxl345.md

If you are using a [Viam Rover](/appendix/try-viam/), this is the accelerometer on it.

> Make sure to physically connect your movement sensor to your machine's computer and turn it on.

Reviewer: ```suggestion
Physically connect your movement sensor to your machine's computer and power both on.
```
